Resources/public/js/views/ez-dashboardblocksview.js

```diff
@@ -152,6 +152,10 @@ YUI.add('ez-dashboardblocksview', function (Y) {
                             bubbleTargets: this,
                             rootLocation: this.get('rootLocation'),
                         }),
+                        new Y.eZ.DashboardBlockMyDraftsView({
+                            priority: 1,
```

Comment:
just to make sure the block are correctly sorted and to let the opportunity for the integrators to put block in between, it's better to let a gap between priorities. For instance, All Content could have a priority of 100 and My Draft a priority of 200.